gluten-core/src/main/scala/io/glutenproject/execution/HashAggregateExecBaseTransformer.scala

```diff
@@ -417,195 +416,47 @@ abstract class HashAggregateExecBaseTransformer(
     var resIndex = index
     val mode = exp.mode
     val aggregateFunc = exp.aggregateFunction
-    aggregateFunc match {
-      case extendedAggFunc
-          if ExpressionMappings.expressionExtensionTransformer.extensionExpressionsMapping.contains(
-            extendedAggFunc.getClass) =>
-        // get attributes from the extended aggregate functions
-        ExpressionMappings.expressionExtensionTransformer
-          .getAttrsForExtensionAggregateExpr(
-            aggregateFunc,
-            mode,
-            exp,
-            aggregateAttributeList,
-            aggregateAttr,
-            index)
```

Comment:
Can not delete this part for the extension, when there are some custom agg functions which have the specified logic to implement.

Reply:
I see that CustomSum should also follow this pattern.

Reply:
CustomSum is only a simple custom ut case, in out internal product, we need to implement some custom logic.

Reply:
Thank you for the explanation. I see. Sorry for removing that portion of the logic.